src/main/java/com/example/smartinfileoperation/SmartinFileOperationApplication.java

```diff
@@ -61,11 +61,12 @@ public static void main(String[] args)  {
 
             for (var entry : ipDnsMap.entrySet()) {
                 System.out.println(entry.getKey());
+                System.out.println(entry.getValue());
                 ps.print(entry.getValue() + " " + entry.getKey());
                 ps.println();
             }
 
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
for getting specific exception, you have to use IOException. Its is not Exception because It is not general error.